wgengine/magicsock/magicsock.go

```diff
@@ -1300,13 +1315,25 @@ func wgRecvAddr(e conn.Endpoint, ipp netaddr.IPPort, addr *net.UDPAddr) *net.UDP
 	return ipp.UDPAddr()
 }
 
+// noteRecvActivity calls the magicsock.Conn.noteRecvActivity hook if
+// e is a discovery-capable peer and it's been awhile since we last
```

Comment:
awhile -> a while

The docstring suggests this function is doing the time-based filtering, but it's not. Either function or docstring needs tweaking

Reply:
Simplified comment.